helm/cluster-cloud-director/templates/cleanup-helmreleases-hook-job.yaml

```
@@ -124,3 +125,26 @@ spec:
             limits:
               memory: "256Mi"
               cpu: "100m"
+{{- if not .Values.connectivity.network.allowAllEgress }}
```

i feel like this belongs under the `internal` section - it doesn't need to be exposed to customers. and also the `connectivity`  field is more about the connectivity of the cluster resources.

also, we can always create this policy regardless of whether the deny-all policies are created - it just won't have any effect if they aren't created.

```suggestion
{{- if .Values.ciliumNetworkPolicy.enabled }}
```

---

to be honest we could just drop the `if` and always create the CNP regardless - makes things more simple

---

That wouldn't be aligned with [your latest PR on Falco](https://github.com/giantswarm/falco-app/pull/224/files) though ?

---

no agreed, but that's not a cluster chart - as i understood it the cluster charts hide non customer-facing stuff under the `internal` key

---

Good point, in that case it should be fine indeed

---

Applied in https://github.com/giantswarm/cluster-cloud-director/pull/248/commits/93124f2e16f09e0b82ebc85900d9b7513c1b29c3